src/components/icons/_icons-mixin.scss

```diff
@@ -3,51 +3,43 @@
     fill: inherit;
   }
 
-  &--light { //default
-    fill: $white;
+  &--icon-black {
```

Comment:
Just wondering if we could skip 'icon' part in class names when we know this is icon component? On the other hand it's can be easier to match it to actual color name in figma.

Reply:
It would change our classnames construction convention https://github.com/brainly/style-guide/blob/214bb15c4251249305460a1f707de1169a5aec41/src/components/icons/Icon.jsx#L539

Reply:
Yes, then function should be adjusted to produce shorted class names. Reason for that was just some word duplication in class name. Now it's `sg-icon--icon-black`, but could be shorter `sg-icon--black`. Nevermind, I think it's not worth changing 😄